modules/ROOT/pages/troubleshoot-guide.adoc

```diff
@@ -45,24 +45,12 @@ tcp://dias-ingestor-nginx.prod.cloudhub.io:5044 ✔
  
 If you have outbound connectivity issues that prevent Runtime Fabric from reaching any of the required Anypoint control plane services, work with your network team to verify that you have completed the required whitelisting as described in xref:install-port-reqs#port-ips-and-hostnames-to-whitelist[Port IPs and Hostnames to Whitelist].
 
-== Resuming a Failed Installation
+== Troubleshoot Cluster Issues
```

Comment:
I wonder if should preface this with something like “If instructed by support” or “when filing a support case”.  The diagnostics dump by itself is not intended to help a customer troubleshoot by themselves.

Reply:
Support will ask the customer to generate either the regular report or the appliance report. Maybe this can be worded more like describing to the customer what is included in each report.

The customer shouldn’t have to feel like they have to choose which command to run

Reply:
There is also https://docs.mulesoft.com/runtime-fabric/latest/using-opscenter#download-debug-info which is ops-center specific but also could use a link to here maybe.

Reply:
@jeff-1amstudios Thank you! Great input - I've updated this. Please see if the updated text works.